tests/test_io.py

```diff
@@ -636,6 +636,217 @@ def test_get_description(self):
         res = ResultDymolaBinary('CoupledClutches_result.mat')
 
         assert res.description[res.get_variable_index("J1.phi")] == "Absolute rotation angle of component"
+    
+    @testattr(stddist = True)
+    def test_modified_result_file_data_diagnostics(self):
```

Comment:
Not mandatory but I prefer if we can add a docstring to the test explaining what the test does. I think it is good for maintenance. Same comment for all the tests below.